ecms_base/ecms_base.install

```diff
@@ -50,8 +50,31 @@ function ecms_base_install() {
 
   $shortcut->save();
 
-  // Call the workflow service to update configuration.
-  \Drupal::service('ecms_workflow.bundle_create')
-    ->assignWorkflowToActiveTypes();
+  // Add correct permissions for the nodes/taxonomies.
+  /** @var \Drupal\ecms_workflow\EcmsWorkflowBundleCreate $workflowBundleCreate */
+  $workflowBundleCreate = \Drupal::service('ecms_workflow.bundle_create');
 
+  /** @var \Drupal\Core\Entity\EntityTypeBundleInfoInterface $bundleInfo */
+  $bundleInfo = \Drupal::service('entity_type.bundle.info');
+
+  $nodes = $bundleInfo->getBundleInfo('node');
+
+  // Guard again an empty array of nodes.
+  if (!empty($nodes)) {
+    // Add the correct workflow to the node types.
+    foreach (array_keys($nodes) as $type) {
+      $workflowBundleCreate->addContentTypeToWorkflow($type);
```

Comment:
This looks like it will pass the 'notification' content type to the workflow assignment, which we don't want. We should be able to use the 'excluded_types' constant in the bundle create class to guard against this as it does in assignWorkflowToActiveTypes

Reply:
I added the content type guard to the `addContentTypeToWorkflow` method.